Anlab.Mvc/AnlabMvc.csproj

```diff
@@ -1,152 +1,137 @@
-    <None Remove="yarn.lock" />
-  </ItemGroup>
-  <ItemGroup>
-    <Content Include="Client\media\icecream.svg">
-      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
-    </Content>
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="wwwroot\Images\Error.png" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Anlab.Core\Anlab.Core.csproj" />
-  </ItemGroup>
-  <ItemGroup>
-    <TypeScriptCompile Include="Client\components\DateSampled.tsx" />
-    <TypeScriptCompile Include="Client\components\TestInfo.tsx" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Update="Pages\**\*.md">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Update="Resources\QueryResource.Designer.cs">
-      <DesignTime>True</DesignTime>
-      <AutoGen>True</AutoGen>
-      <DependentUpon>QueryResource.resx</DependentUpon>
-    </Compile>
-  </ItemGroup>
-  <ItemGroup>
-    <Content Update="Views\Reviewer\Details.cshtml">
-      <Pack>$(IncludeRazorContentInPack)</Pack>
-    </Content>
-    <Content Update="Views\Shared\_PrintScript.cshtml">
-      <Pack>$(IncludeRazorContentInPack)</Pack>
-    </Content>
-    <Content Update="Views\Shared\_PrintStyle.cshtml">
-      <Pack>$(IncludeRazorContentInPack)</Pack>
-    </Content>
-    <Content Update="Views\Shared\_DataTableFixedHeaderStylePartial.cshtml">
-      <Pack>$(IncludeRazorContentInPack)</Pack>
-    </Content>
-    <Content Update="Views\Shared\_DataTableFixedHeaderScriptsPartial.cshtml">
-      <Pack>$(IncludeRazorContentInPack)</Pack>
-    </Content>
-    <Content Update="Views\Templates\_OrderPartialResults.cshtml">
-      <Pack>$(IncludeRazorContentInPack)</Pack>
-    </Content>
-    <Content Update="wwwroot\Images\anlab.jpeg">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Update="wwwroot\Images\anlab170.png">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
+<Project ToolsVersion="15.0" Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <TypeScriptCompileBlocked>true</TypeScriptCompileBlocked>
+    <IsPackable>false</IsPackable>
+    <UserSecretsId>f3c999a5-2304-44e0-a49e-f43333bf9ccf</UserSecretsId>
+    <Version>1.1.0.0</Version>
+    <Description>Order and results management system for AnLab@UCDavis</Description>
+    <TypeScriptToolsVersion>2.3</TypeScriptToolsVersion>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DebugType>full</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="AspNetCore.Security.CAS" Version="2.0.1" />
+    <PackageReference Include="AzureActiveDirectorySearcher" Version="1.0.0" />
+    <PackageReference Include="Dapper" Version="1.50.2" />
+    <!-- <PackageReference Include="Humanizer.Core" Version="2.2.0" /> -->

```

Comment:
Is this not needed for `id.Humanize(LetterCasing.Title);` ?

Reply:
Don't recall exactly what the issue was. I think it was conflicting with a transitive dependency, so I decided to just let it use the one referenced by whatever other package is using it.